local-dev.yml

```diff
@@ -113,10 +111,29 @@ services:
       - "8983:8983"
     stdin_open: true
     tty: true
+  nginx:
+    build:
+      context: ./nginx
+      dockerfile: Dockerfile-nginx
+    hostname: nginx
+    container_name: nginx
+    links:
+      - hydroshare:hydroshare
+    depends_on:
+      - hydroshare
+    ports:
+      - "80:80"
+    volumes:
+      - "./log/nginx:/var/log/nginx"
+      - ".:/hydroshare"
+      - "temp_vol:/hs_tmp"
+      - "share_vol:/irods/cache"
+      - "idata_vault_vol:/irods/data"
+      - "iuser_vault_vol:/irods/user"
   hydroshare:
     build:
       context: .
-      dockerfile: Dockerfile-hydroshare
+      dockerfile: Dockerfile
     container_name: hydroshare
```

Comment:
ideally every dockerfile will differentiate itself and there will be none named just 'Dockerfile' in the entire project

Reply:
So what will happen if in future we have to use other hs_docer_base image, modify all Dockerfile???